mkdocs.yml

```diff
@@ -0,0 +1,18 @@
+docs_dir: docs/book
+site_dir: docs/html
+extra:
+  project: Packages
+  current_version: v3
+  versions:
+    - v3
+nav:
+  - Home: index.md
+  - v3:
```

Comment:
Change the order of the elements as follows:
- `Overview`
- `Installation`
- `Configuration`